permission_handler/lib/permission_handler.dart

```diff
@@ -18,13 +18,13 @@ PermissionHandlerPlatform get _handler => PermissionHandlerPlatform.instance;
 /// Opens the app settings page.
 ///
 /// Returns [true] if the app settings page could be opened, otherwise [false].
-Future<bool> openAppSettings() => _handler.openAppSettings();
+Future<bool?> openAppSettings() => _handler.openAppSettings();
```

Comment:
I think this should be allowed to be `null`. The method should always return one of the following values:

- `true` if the App settings could be opened;
- `false` if the App settings could not be opened;
-  Or an exception if something went wrong.

So we need to handle this in the method channel implementation.